### PR TITLE
Add players hp to store

### PR DESCRIPTION
**Pull request by AntonPankou**

### CODE REVIEW

File: client/src/app/app.module.ts [hunk 1]
```diff
@@ -43,7 +43,10 @@ import {
   CardsEffects,
   reducers,
   CardsFacade,
-  initialState
+  initialState,
+  playersHPinitialState,
+  PlayersHPActions,
+  PlayersHPEffects
```
Comment (vharadkou) [inline client/src/app/app.module.ts]: prefer import from store/feature; like `import { PlayersHPEffects, PlayersHPActions } from 'store/players'`

File: client/src/store/players-hp/players-hp.effects.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import { Action } from '@ngrx/store';
+
+import * as playersHPActions from './players-hp.action';
+import { PlayersHPState } from './interfaces';
+
+@Injectable()
+export class PlayersHPEffects {
+  public baseUrl = 'http://www.mocky.io/v2/5bf697823200008e005d0ff1';
+
+  @Effect() public getPlayersHP$: Observable<Action> = this.actions$.pipe(
```
Comment (vharadkou) [inline client/src/store/players-hp/players-hp.effects.ts]: rename get to load

File: client/src/app/app.module.ts (hunk 1 above)
Comment (AntonPankou) [reply client/src/app/app.module.ts]: Fixed

File: client/src/store/players-hp/players-hp.effects.ts (hunk 2 above)
Comment (AntonPankou) [reply client/src/store/players-hp/players-hp.effects.ts]: Fixed

**Pull request merged**